client/src/pages/SignUp.js

```diff
@@ -0,0 +1,243 @@
+        break;
+    }
+  };
+
+  loadSignIn = () =>
+    ReactDOM.render(<SignIn />, document.getElementById("root"));
+
+  render() {
+    const { classes } = this.props;
+    return (
+      <div className={classes.signUpContainer}>
+        <div className={classes.leftSideContainer}>
+          <div className={classes.titleLogo}>
+            <Typography variant="h4">CHEF'S MENU</Typography>
+          </div>
+          <Card className={classes.card}>
+            <CardContent>
+              <Typography className={classes.title} gutterBottom>
+                Create an account
+              </Typography>
+              <form autoComplete="off">
+                <Box display="flex">
+                  <TextField
+                    className={classes.textField}
+                    required
+                    autoFocus
+                    id="signupName"
+                    name="signupName"
+                    label="Name"
+                    placeholder="Enter your name"
+                    error={this.state.usernameError !== "" ? true : false}
+                    variant="outlined"
+                    onChange={this.validateChange}
+                    helperText={this.state.usernameError}
+                  />
+                </Box>
+                <Box display="flex">
+                  <TextField
+                    className={classes.textField}
+                    required
+                    name="signupEmail"
+                    label="Email"
+                    placeholder="Enter your e-mail address"
+                    variant="outlined"
+                    error={this.state.emailError !== "" ? true : false}
+                    helperText={this.state.emailError}
+                    onChange={this.validateChange}
+                  />
+                </Box>
+                <Box display="flex">
+                  <TextField
+                    className={classes.textField}
+                    required
+                    type="password"
+                    name="signupPwd"
+                    label="Password"
+                    error={this.state.passwordError !== "" ? true : false}
+                    placeholder="Enter password"
+                    variant="outlined"
+                    helperText="Minimum 6 characters"
+                    onChange={this.validateChange}
+                  />
+                </Box>
+                <Box display="flex">
+                  <TextField
+                    className={classes.textField}
+                    required
+                    type="password"
+                    name="signupPwdConfirm"
+                    label="Confirm Password"
+                    placeholder="Re-enter password"
+                    variant="outlined"
+                    onChange={this.validateChange}
+                    error={
+                      this.state.passwordConfirmError === "Mismatch"
+                        ? true
+                        : false
```

Comment:
Let's keep your code uniform. In all the other cases above you were checking `!==""`. Do the same here

Reply:
Done.